Appraisals

```diff
@@ -1,23 +1,17 @@
-appraise 'rails-5' do
-  gem 'actionpack', '~> 5.0.2'
-  gem 'activeresource', '~> 5.0.2'
-  gem 'activesupport', '~> 5.0.2'
+appraise 'rails-52' do
```

Comment:
FYI: Looks like `Appraisals` was outdated, so I fix this and add support Rails versions from 5.2 till 6.1.